core/blockchain.go

```diff
@@ -1369,6 +1376,15 @@ func (self *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 		go self.eventMux.Post(RemovedLogsEvent{deletedLogs})
 	}
 
+	if len(oldChain) > 0 {
+		fmt.Println("side chain length =", len(oldChain))
```

Comment:
Ah, I think that was a debug log for Jeff to find the test issue. Wasn't in the code before :D :P